### ISSUE TITLE
eslint-plugin-putout: comments break location reporting

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#22]
Example:

```js
// comment
foo;
```

```sh
eslint .
```

Expected result: reported line number should be `2`

```text
2:1  error  Unused expression statement (remove-unused-expressions)  putout/putout
```

Actual result: reported line number is `1`

```text
1:1  error  Unused expression statement (remove-unused-expressions)  putout/putout
```

**Comment by EvgenyOrekhov** [comment]
If the code doesn't have comments, location reporting works as expected.

**Comment by coderaiser** [comment]
Thanks a lot :), fixed with d7c6640, landed in `eslint-plugin-putout v5.2.0` 🎉 . Is it works for you?

**Comment by EvgenyOrekhov** [comment]
Yes, it works correctly now, thanks!
